### PR TITLE
Adding scripts for adobe analytics

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

File: src/pug/head.pug [hunk 1]
```diff
@@ -3,3 +3,4 @@ meta(name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit
 base(href=`${ release === 'insights' ? '/' : '/beta/'}`)
 link(rel='stylesheet', href='https://use.fontawesome.com/releases/v5.0.10/css/all.css', integrity='sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg', crossorigin='anonymous')
 link(rel='stylesheet', href='./apps/chrome/chrome.css')
+script(id="adobe_dtm" src="//www.cloud.redhat.com" type="text/javascript")
```
Comment (iphands) [inline src/pug/head.pug]: Lol wat dis?
src="//www.cloud.redhat.com" ? That aint no script. Its not even a valid URL.

Comment (ryelo) [reply src/pug/head.pug]: From the mojo:

```
<script id="adobe_dtm" src="{DPAL URL}" type="text/javascript"></script>

DPAL Production URLs:
//www.redhat.com/dtm.js - Sends data to the "Red Hat Catchall" report suite in Adobe Analytics
``

Comment (ryelo) [reply src/pug/head.pug]: Oh nevermind. I didn't push up the change for this

Comment (ryelo) [reply src/pug/head.pug]: @iphands Unstaged commits. I was debugging to see if I could get around the catchall

Comment (ryelo) [reply src/pug/head.pug]: @iphands updated

**Pull request merged**